package-lock.json

```diff
@@ -13749,6 +13756,7 @@
         "@tech-matters/hrm-types": "^1.0.0",
         "@types/deep-diff": "^1.0.1",
         "@types/express": "^4.17.13",
+        "@types/http-errors": "*",
```

Comment:
Is it weird that the version range generated here is `*` even though you set `^2.0.1` in the `package.josn`?

Reply:
Should we sorry about this? Is just a dev dep for the types.

Reply:
I'm just asking a Q, no need to change anything